CHANGELOG.md

```diff
@@ -51,7 +51,7 @@
 * [#3485](https://github.com/bbatsov/rubocop/issues/3485): Make OneLineConditional cop not register offense for empty else. ([@tejasbubane][])
 * [#3508](https://github.com/bbatsov/rubocop/pull/3508): Fix false negatives in `Rails/NotNullColumn`. ([@pocke][])
 * [#3462](https://github.com/bbatsov/rubocop/issues/3462): Don't create MultilineMethodCallBraceLayout offenses for single-line method calls when receiver spans multiple lines. ([@maxjacobson][])
-
+[#3514](https://github.com/bbatsov/rubocop/issues/3514) Changed the regular expression which defines normal case. ([@b-t-g])
```

Comment:
Do the most recent updates fix everything you had in mind, or does anything need to be adjusted further?